### ISSUE TITLE
bug when clicking on the leapmotion object

### ISSUE THREAD

**Issue by mxa** [chikashimiyama/Pd_leapmotion#3]
When clicking on the leapmotion object, a window pops up that can't be closed. Is this maybe something that comes from flext? I have never seen this kind of window before.
![screenshot from 2017-10-17 17-43-02](https://user-images.githubusercontent.com/1220707/31769201-36c8d0be-b505-11e7-8e20-5d0eaeffae15.png)
This window takes the focus, so Pd becomes unusable and needs to be killed.
To avoid this, never click on the leapmotion object.

**Comment by grrrr** [comment]
Yes, it is a flext window for showing instance attributes - the problem can be resolved by recompiling the leapmotion object.

**Comment by mxa** [comment]
@grrrr you mean it should be part of the manual to build it twice and the second time removes this strange behaviour?
